nucypher/characters/lawful.py

```diff
@@ -16,41 +16,44 @@
 """
 
 import random
+import socket
 from collections import OrderedDict
+from functools import partial
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Set
 
 import maya
 import requests
-import socket
 import time
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurve
 from cryptography.hazmat.primitives.serialization import Encoding
 from cryptography.x509 import load_pem_x509_certificate, Certificate, NameOID
 from eth_utils import to_checksum_address
-from functools import partial
 from twisted.internet import threads
-from typing import Dict
-from typing import Iterable
-from typing import List
+from umbral.keys import UmbralPublicKey
+from umbral.signing import Signature
 
+from bytestring_splitter import BytestringKwargifier, BytestringSplittingError
 from bytestring_splitter import BytestringSplitter, VariableLengthBytestring
 from constant_sorrow import constants
-from constant_sorrow.constants import PUBLIC_ONLY
+from constant_sorrow.constants import INCLUDED_IN_BYTESTRING, constant_or_bytes
```

Comment:
This results in a `ValueError`, I believe, with the latest version of constant sorrow:

```
E   ValueError: Use ALL_CAPS names for constants.  See https://www.python.org/dev/peps/pep-0008/#constants.
```

Reply:
Umm yeah, it's probably defined in the wrong place.  It doesn't belong in `constants`.

But indeed, it's a utility function, and a pretty interesting one if I do say so myself.  :-)

Reply:
In any case I cannot run the tests with the latest version of constant sorrow, due to this.